### PR TITLE
Cross-build for Native

### PR DESCRIPTION
**Pull request by armanbilge**
Squashes https://github.com/typelevel/cats-parse/pull/376 and brings it across the finish line.

### CODE REVIEW

**Review by johnynek — APPROVED** [review]
Thanks for finishing this up!

**Comment by armanbilge** [comment]
Hmm, seems that the merge commit https://github.com/typelevel/cats-parse/commit/edd465d8901739b067a8e6d1f90acec46de6b6c0 lost the attribution to @i10416 in https://github.com/typelevel/cats-parse/pull/426/commits/cdd3d4a97e30d9aa3d524daf2655e19980f8d0d1 ... I'm so sorry about that 😕

File: project/plugins.sbt [hunk 1]
```diff
@@ -1,3 +1,5 @@
+addSbtPlugin("org.portable-scala" % "sbt-scala-native-crossproject" % "1.2.0")
+addSbtPlugin("org.scala-native" % "sbt-scala-native" % "0.4.4")
```
[the remark this answers is not in the document]
Comment (armanbilge) [reply project/plugins.sbt:2]: Ah, I forgot update this. No problem, there will be a steward update here shortly :)

[the remark this answers is not in the document]
Comment (armanbilge) [reply project/plugins.sbt:2]: Actually already is!
- https://github.com/typelevel/cats-parse/pull/427

**Comment by i10416** [comment]
> Hmm, seems that the merge commit [edd465d](https://github.com/typelevel/cats-parse/commit/edd465d8901739b067a8e6d1f90acec46de6b6c0) lost the attribution to @i10416 in [cdd3d4a](https://github.com/typelevel/cats-parse/commit/cdd3d4a97e30d9aa3d524daf2655e19980f8d0d1) ... I'm so sorry about that 😕

No worries. Thanks for taking over my PR. I was a bit occupied :(

**Pull request merged**